prover_rust/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "prover_rust"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+
+
+
+[patch.crates-io]
+ethers-signers  = { git = "https://github.com/scroll-tech/ethers-rs.git", branch = "v2.0.7" }
```

Comment:
why not call it `ethers-rs`?

Reply:
haha it is like the following js code

```
let { key1 } = obj1;
let { key2 } = obj2;
```

the "ethers-signers" is one of the packages inside https://github.com/scroll-tech/ethers-rs.git

Reply:
yeah the ethers-rs project uses cargo workspace having multiple sub packages.
the "ethers-signers" is one of the package which is exactly need